Apps/Playground/Scripts/experience.js

```diff
@@ -219,7 +220,33 @@ CreateBoxAsync().then(function () {
                     });
                 }
 
-                xr.baseExperience.enterXRAsync(vr ? "immersive-vr" : "immersive-ar", "unbounded", xr.renderTarget);
+                let sessionMode = vr ? "immersive-vr" : "immersive-ar"
+                if (hololens) {
+                    // Because HoloLens 2 is a head mounted display, its Babylon.js immersive experience more closely aligns to vr
+                    sessionMode = "immersive-vr";

```

Comment:
We could inline this above as
`// Because HoloLens 2 is a head mounted display, its Babylon.js immersive experience more closely aligns to vr
let sessionMode = (vr || hololens) ? "immersive-vr" : "immersive-ar"`

Reply:
I don't have a strong attachment to how this is currently written, but my personal preference would be to have things related to hololens clearly called out in their own code stub for legibility. What do others think?